### PR TITLE
feat: add an intermediate class without Validation

### PR DESCRIPTION
**Pull request by raszi**
Closes #96

### CODE REVIEW

**Comment by raszi** [comment]
@andyjeffries did you have a change to look at this? Do you need me to change something else?

**Comment by andyjeffries** [comment]
Thanks for this. The only things I would say would be it would be great if we could spec that the ActiveModel stuff works if integrated in to a custom base class that uses this new `BaseWithoutValidation`. Also, a doc to show users how to use it. Thanks for your effort. if we can get those last couple of bits in there, I'll merge and release it.

**Comment by raszi** [comment]
Thank you @andyjeffries. I am working on the documentation and the examples.

**Comment by raszi** [comment]
I believe I added everything what you asked for. Could you please check it @andyjeffries, thanks!

File: docs/validation.md [hunk 1]
```diff
@@ -83,6 +84,34 @@ The following attributes will pass validation since they explicitly `allow_nil`:
 - `:retirement_age`
 - `:favorite_authors`
 
+## ActiveModel::Validations
+
+This built-in validations have a bit different syntax than the ActiveModel validations and use a different codebase.
+
+You can opt-out from the built-inv validations and use the `ActiveModel` validations instead if you inherit from the `Flexirest::BaseWithoutValidation` class instead of the `Flexirest::Base`.
```
Comment (andyjeffries) [inline docs/validation.md]: `inv` -> `in`

**Comment by andyjeffries** [comment]
Aside from that one typo, happy to merge it. Let me know and I'll merge it and push a new release. Thanks for your work.

**Comment by raszi** [comment]
Fixed, thanks.

**Comment by andyjeffries** [comment]
Pushed as v1.6.8. Thank you!

**Comment by raszi** [comment]
Thank you! 🍾

**Pull request merged**